airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/build_image/steps/manifest_only_connectors.py

```diff
@@ -43,11 +43,20 @@ async def _build_from_base_image(self, platform: Platform) -> Container:
         """
         self.logger.info(f"Building connector from base image in metadata for {platform}")
 
+        # Mount manifest file
         base_container = self._get_base_container(platform).with_file(
             f"source_declarative_manifest/{MANIFEST_FILE_PATH}",
             (await self.context.get_connector_dir(include=[MANIFEST_FILE_PATH])).file(MANIFEST_FILE_PATH),
         )
 
+        # Mount components file if it exists
+        components_file = self.context.connector.code_directory / COMPONENTS_FILE_PATH
+        if components_file.exists():
```

Comment:
🧪 Now, can we add a test to assert this works?

You should be able to modify this test
https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/tests/test_build_image/test_manifest_only_connectors.py#L51

And
1. mock it so `components_file.exists()` always returns true
2. We assert base_container.with_file was called with the expected params! (you'll notice we do that with_exec and with_label)

Let me know if you want a hand!